<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ganar</class>
 <widget class="QDialog" name="Ganar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>540</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="opaqueResize">
    <bool>false</bool>
   </property>
   <property name="handleWidth">
    <number>0</number>
   </property>
   <property name="childrenCollapsible">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="volver_a_jugar">
    <property name="styleSheet">
     <string notr="true">border-image: url(:/imagenes2/VOLVER A JUGAR.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="siguiente_nivel">
    <property name="styleSheet">
     <string notr="true">border-image: url(:/imagenes2/NEXT LVL.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="salir_del_juego">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/imagenes2/SALIR DEL JUEGO.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
